From: Michel Normand <normand@linux.vnet.ibm.com>
Subject: sword sysdata changes
Date: Wed, 24 Feb 2016 14:12:06 +0100

sword sysdata changes
required to avoid build error of xyphos package
when built on PowerPC architecture

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>
---
 include/sysdata.h |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

Index: sword-1.8.1/include/sysdata.h
===================================================================
--- sword-1.8.1.orig/include/sysdata.h
+++ sword-1.8.1/include/sysdata.h
@@ -29,8 +29,9 @@
 #ifdef USE_AUTOTOOLS
 #include "config.h"
 #endif
+#include <asm/types.h>
 
-
+#if !defined(_ASM_GENERIC_INT_L64_H) && !defined( _ASM_GENERIC_INT_LL64_H)
 typedef signed char __s8;
 typedef unsigned char __u8;
 
@@ -51,6 +52,7 @@ typedef unsigned __int64 __u64;
 typedef signed long long __s64;
 typedef unsigned long long __u64;
 #endif
+#endif
 
 #undef __swswap16
 #undef __swswap32
